### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by Nickrader**
docs: Added #include <sc2api/sc2_unit_filters.h> to Tutorial3.  Added #include <sc2api/sc2_unit_filters.h> to the 'Full Source Code' and the instructions.

### CODE REVIEW

**Comment by alkurbatov** [comment]
Many thank for the PR.

Small style corrections needed:
a. Please, squash (unite) the commits into single entity as both fix the same topic.
b. Please set the commit title to `docs: Added #include <sc2api/sc2_unit_filters.h> to Tutorial3` etc, i.e. move description of the PR into the commit message. Otherwise the changelog will show only `Update tutorial3.md `.

File: docs/tutorial3.md [hunk 1]
```diff
@@ -140,7 +145,7 @@ Full Source Code
 
 ```C++
 #include <sc2api/sc2_api.h>
-
+#include <sc2api/sc2_unit_filters.h>
```
Comment (alkurbatov) [inline docs/tutorial3.md:148]: According to the Google C++ style guide you should group sc2api imports together. Thus it should be:
```
```C++
#include <sc2api/sc2_api.h>
#include <sc2api/sc2_unit_filters.h>

#include <iostream>
```

**Comment by alkurbatov** [comment]
> Many thank for the PR.
> 
> Small style corrections needed:
> a. Please, squash (unite) the commits into single entity as both fix the same topic.
> b. Please set the commit title to `docs: Added #include <sc2api/sc2_unit_filters.h> to Tutorial3` etc, i.e. move description of the PR into the commit message. Otherwise the changelog will show only `Update tutorial3.md `.

Still actual.

**Comment by Nickrader** [comment]
Going through various docs trying to figure out squashing commits and such.  Hopefully soon ;)

**Comment by Nickrader** [comment]
In progress, it may be awhile.

**Comment by Nickrader** [comment]
I think I figured it out.  My mistake was using the Github Desktop App without knowing what I was doing, and the solution seemed to be to use Git from the command line.

**Comment by Nickrader** [comment]
In retrospect force pushing my branch, may have not been the best solution (I feel as if a comment is now missing), but I was kinda lost after hitting the shiny blue button too many times on the Github Desktop App.  At one point fighting a detached head issue as I played around, I was happy to hack my way into something resembling a fix.

**Comment by alkurbatov** [comment]
Missing comment is ok, it happens frequently because the original code was altered.
Thank you for the fix!

**Comment by Nickrader** [comment]
You are welcome.  Thanks for the guidance.

**Pull request merged**